CORE-DEVELOPMENT-README.md

```diff
@@ -0,0 +1,18 @@
+# CORE DEVELOPMENT GUIDE
+
+1. Clone this repository by using git clone https://github.com/bhanu951/drupal-core-development-project.git
+2. cd repos
+3. git clone --branch=10.1.x https://git.drupalcode.org/project/drupal.git drupal
+4. ddev composer install
+5. ddev composer require drupal/admin_toolbar drupal/devel
+6. ddev phpcs repo/drupal/core/<PATH_TO_FILE> (from project root)
```

Comment:
Could this be added to a section on using DDEV in the main README please?

Reply:
Sure, will update it.

Reply:
Done, Moved to the main README